test/unit/sql/index.test.js

```diff
@@ -232,6 +233,38 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
         });
       });
     }
+
+    if (current.dialect.supports.index.include) {
```

Comment:
this test should run on all dialects, but dialects that don't support it should check that an error is thrown. You can do it like this:

```ts
// arrow function so expectsql can catch the error
expectsql(() => sql.addIndexQuery('User',  {
  name: 'email_include_name',
  fields: ['email'],
  include: ['first_name', 'last_name'],
  unique: true,
}), {
  'mysql mariadb': new Error('Your error message here')
  postgres: 'CREATE UNIQUE INDEX [email_include_name] ON [User] ([email]) INCLUDE ([first_name], [last_name])',
});
```

Reply:
Done